extensions/opentelemetry/opentelemetry-exporter-jaeger/runtime/src/main/java/io/quarkus/opentelemetry/exporter/jaeger/runtime/JaegerSubstitutions.java

```diff
@@ -0,0 +1,40 @@
+package io.quarkus.opentelemetry.exporter.jaeger.runtime;
+
+import static java.util.Objects.requireNonNull;
+
+import javax.net.ssl.SSLException;
+import javax.net.ssl.X509TrustManager;
+
+import com.oracle.svm.core.annotate.Substitute;
+import com.oracle.svm.core.annotate.TargetClass;
+
+import io.grpc.ManagedChannelBuilder;
+import io.grpc.netty.GrpcSslContexts;
+import io.grpc.netty.NettyChannelBuilder;
+import io.opentelemetry.exporter.otlp.internal.grpc.ManagedChannelUtil;
+
+final class JaegerSubstitutions {
```

Comment:
Any change you could add a comment with the purpose of this substitution?

Reply:
Sure.